### PR TITLE
Add Python 3.12

### PR DESCRIPTION
**Pull request by Luke-Roy-IBM**

### CODE REVIEW

**Comment by Luke-Roy-IBM** [comment]
This PR will address issue https://github.com/apache/openwhisk-runtime-python/issues/149

File: core/python312Action/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+ADD bin/compile /bin/compile
+ADD lib/launcher.py /lib/launcher.py
+
+# log initialization errors
+ENV OW_LOG_INIT_ERROR=1
+# the launcher must wait for an ack
+ENV OW_WAIT_FOR_ACK=1
+# using the runtime name to identify the execution environment
+ENV OW_EXECUTION_ENV=openwhisk/action-python-v3.11
```
Comment (dgrove-oss) [inline core/python312Action/Dockerfile:66]:
```suggestion
ENV OW_EXECUTION_ENV=openwhisk/action-python-v3.12
```

File: core/python312Action/requirements.txt [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# default packages available for action-python-v3.11
```
Comment (dgrove-oss) [inline core/python312Action/requirements.txt:1]:
```suggestion
# default packages available for action-python-v3.12
```

**Review by dgrove-oss — CHANGES_REQUESTED** [review]
Two small fixes; otherwise looks good.

**Comment by Luke-Roy-IBM** [comment]
@dgrove-oss fixes applied 👍

**Review by dgrove-oss — APPROVED** [review]
lgtm

**Pull request merged**